packages/kit/src/vite/dev/index.js

```diff
@@ -189,7 +190,7 @@ export async function dev(vite, svelte_config) {
 
 		remove_html_middlewares(vite.middlewares);
 
-		vite.middlewares.use(async (req, res) => {
+		vite.middlewares.use(async (req, res, next) => {
```

Comment:
didn't end up using this after all

```suggestion
		vite.middlewares.use(async (req, res) => {
```